test/hotspot/jtreg/vmTestbase/nsk/jdi/ClassPrepareEvent/referenceType/refType001.java

```diff
@@ -130,8 +133,10 @@ public void run() {
                             Event event = eventIterator.nextEvent();
 //                            log.display("\nEvent received:\n  " + event);
```

Comment:
Nit: Would it make sense to uncomment this log message? How many events are normally printed?